lcls2-cc-lib/Library/POUs/FB_L2SI_Flowmeter.TcPOU

```diff
@@ -1,22 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <POU Name="FB_L2SI_Flowmeter" Id="{f4fe6025-68b2-43d4-8493-5a0f33999a1a}" SpecialFunc="None">
-    <Declaration><![CDATA[FUNCTION_BLOCK FB_L2SI_Flowmeter
-VAR
-    {attribute 'pytmc' := '
-        pv: MA
-        io: input
-    '}
-    fRaw AT %I*: INT;
-
-    {attribute 'pytmc' := '
-        pv: FLOW
```

Comment:
I did some checks and we never used these PVs in pcdsdevices, so it should be good to jettison them.